source/type.cpp

```diff
@@ -461,14 +461,34 @@ std::string standard_name(clang::QualType const &qt)
 	//qt.dump();
 	string r = qt.getAsString();
 
-	// if( r == "std::array::size_type" ) {
-	// //if( begins_with(r, "std::array<" ) ) {
-	// 	outs() << "--> " << r << '\n';
-	// 	qt.dump();
-	// }
 
 	// if( begins_with(r, "std::") ) return r; //standard_name(r);
-	static std::set<string> standard_names {"std::size_t"};
+	static std::set<string> standard_names;
+	
+	if (standard_names.empty())
+	{
+		standard_names =
+		{
+			// <cstddef>
+			"std::size_t", "std::ptrdiff_t",
+
+			// <stddef.h>
+			"size_t", "ptrdiff_t",
+
+			// <cstdint> (most common ones only)
+			"std::int8_t", "std::int16_t", "std::int32_t", "std::int64_t",
+			"std::uint8_t", "std::uint16_t", "std::uint32_t", "std::uint64_t",
+			"std::intmax_t", "std::uintmax_t", "std::intptr_t", "std::uintptr_t",
+
+			// <stdint.h> (most common ones only)
+			"int8_t", "int16_t", "int32_t", "int64_t",
+			"uint8_t", "uint16_t", "uint32_t", "uint64_t",
+			"intmax_t", "uintmax_t", "intptr_t", "uintptr_t",
+		};
+		
+		for (const auto &t : Config::get().standard_types)
```

Comment:
pedantic and optional: would be good if project-standard placement of `const` is used, ie `auto const &`

Reply:
sure, done now.